include/superior_mysqlpp/low_level/dbdriver.hpp

```diff
@@ -319,23 +320,54 @@ namespace SuperiorMySqlpp { namespace LowLevel
             return escapeStringNoConnection(original.c_str(), original.length());
         }
 
+        // XXX: makeHexString functions can be implemented by one function using constexpr-if from c++17
```

Comment:
Please rename XXX to TODO or something more describing the purpose of the comment

Reply:
XXX is IMHO quite standard, it can be found in Linux, Python standard library or GNU C++ library.